docs/machine-learning/ml-problem-formulation-envisioning.md

```diff
@@ -1,76 +1,92 @@
-# Envisioning and Problem Formulation
+# Discovery
+##### Envisioning and Problem Formulation
 
 Before beginning a data science investigation, we need to define a problem statement which the data science team can explore; this problem statement can have a significant influence on whether the project is likely to be successful.
 
-## Envisioning goals
+## Discovery goals
 
-The main goals of the envisioning process are:
+The main goals of the discovery process are:
+*	Establish existing understanding of the problem domain and the underlying business objectives
```

Comment:
The information is great, so thank you for the contribution. 

In general, however, this text is a bit heavy to read - can we rewrite this in a bit more "Plain English" (see [http://www.plainenglish.co.uk/how-to-write-in-plain-english.html](http://www.plainenglish.co.uk/how-to-write-in-plain-english.html) for tips) to make sure our intentions come across correctly? 

Examples of a few potential rewordings

- Understand the problem domain, and business objectives
- (? Not completely sure what this sentence means, but perhaps) State the limits of the problem?
- Explain how potential solutions may interface with existing code, and how we can measure the performance of the suggested solutions.
- Determine what data is available, and the origin and quality of the data. Take special care to understand if the data may be biased.
- ...

Similar with the rest of the document.

Reply:
@TessFerrandez Noted, I will be able to get to this around the holidays